docs/source/features/dataset_management.rst

```diff
@@ -74,11 +74,13 @@ Two types of datasets can be read from TensorBay:
 .. _my datasets(or team datasets): https://gas.graviti.cn/tensorbay/dataset-list
 .. _TensorBay: https://gas.graviti.cn/tensorbay/
 
+Take the :ref:`Uploading of BSTLD <examples/bstld:Read Dataset>` as an example.
```

Comment:
The commit message says `format cross-references`, but this cross-references is an addition. Maybe you need to update your commit message~